public/lessonBuilder/lessonItemsController.js

```diff
@@ -103,14 +103,22 @@ angular.module('newApp')
         modules.get($routeParams.id, $routeParams.id2)
         .success(function(module){
             $scope.moduleName = module.name;
+            $scope.module = module;
             moduleItems.get($routeParams.id, $routeParams.id2, pageNum)
             .success(function (lessonItems, status, headers) {
-                var pgs = makePageMap(headers('link'));
+               var pgs = makePageMap(headers('link'));
+
+               countItemTypes(lessonItems);
+               $scope.pages = pgs;
+               $scope.pageNumbers = createArray(pgs.last);
+               $scope.lessonItems = lessonItems;
+
+               moduleMetadata.get($routeParams.id2)
+               .success(function(meta) {
+                 $scope.module.learningObjectives = (meta ? meta : []);
+
+                });
```

Comment:
I don't see it.  Perhaps you referring to the space difference between the deleted and added code above?